docs/content/integrations/dbt/reference.mdx

```diff
@@ -390,6 +391,44 @@ def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
     yield from dbt.cli(["build"], context=context).stream()
 ```
 
+### Customizing tags
+
+For dbt models, seeds, and snapshots, the default Dagster tags will be the dbt node's configured tags. dbt tags are strings, but Dagster tags are key-value pairs. To bridge this divide, the dbt tag string is used as the Dagster tag key, and the Dagster tag value is set to special sentinel value `"__dagster_no_value"`.
```

Comment:
Let's hoist this up to a `Note`, and reorder the first sentence so that we don't start with lowercase "dbt".
```suggestion
<Note>
In Dagster, tags are key-value pairs. However, in dbt, tags are strings. To bridge this divide, the dbt tag string is used as the Dagster tag key, and the Dagster tag value is set to special sentinel value `"__dagster_no_value"`.

Any dbt tags that don't match Dagster's supported tag key format (e.g. they contain unsupported characters) will be ignored by default.
</Note>

For dbt models, seeds, and snapshots, the default Dagster tags will be the dbt node's configured tags.
```